CaloCluster/fcl/prolog_trigger.fcl

```diff
@@ -40,16 +43,34 @@ CaloClusterFast :
     includeCrystalHits    : true
 }
 
+NewCaloClusterFast :
+{
+  module_type            : CaloProtoClusterFromCrystalHit
+  caloCrystalModuleLabel : FastCaloCrystalHitFromHit
+  mainClusterCollName    : None
+  splitClusterCollName   : None
+  isOnline               : true
+  minClusterEnergy       : 50
+  EminSeed               : 20 
+  ExpandCut              : 1 
+  EnoiseCut              : 1
+  timeCut                : @local::HitMakerBlindTime   
+  deltaTime              : 5.01
+  diagLevel              : 0
+}  
 
 
 CaloClusterTrigger :
 {
    producers : {
        CaloClusterFast  : @local::CaloClusterFast
        CaloTrigger      : @local::CaloTrigger
+       NewCaloClusterFast  : @local::NewCaloClusterFast
    }
 
    Reco : [ CaloClusterFast ]
+   NewReco : [ NewCaloClusterFast ]
+   CrystalReco : [FastCaloRecoDigiFromDigi, FastCaloCrystalHitFromHit]
```

Comment:
This line is duplicated from CaloReco/fcl/prolog_trigger.fcl
I think you should define the NewReco sequence like: NewReco : [ @sequence::OnlineCaloReco.Reco, NewCaloClusterFast ]